$MirOS: ports/www/dillo/v2/patches/patch-src_IO_about_c,v 1.2 2010/02/21 20:08:42 bsiegert Exp $
--- src/IO/about.c.orig	Wed Jul  1 18:24:42 2009
+++ src/IO/about.c	Mon Aug 24 16:30:04 2009
@@ -284,8 +284,8 @@ const char *const AboutSplash=
 "2.1:<br>\n"
 "<ul>\n"
 "<li>Implemented basic CSS infrastructure!\n"
-"<li>Read user style from ~/.dillo/style.css.\n"
-"<li>Added configurable keybindings! (in ~/.dillo/keysrc)\n"
+"<li>Read user style from ~/.etc/dillo/style.css.\n"
+"<li>Added configurable keybindings! (in ~/.etc/dillo/keysrc)\n"
 "<li>Implemented \"search previous\" in string searches.\n"
 "<li>Ported the command line interface from dillo1\n"
 "<li>Set middle click to submit in a new TAB. (Helps to keep form data!)\n"
@@ -354,7 +354,7 @@ const char *const AboutSplash=
 "   <a href='http://www.dillo.org/dillorc'>dillorc</a>\n"
 "   (readable  config)  file within the tarball; It is well-commented\n"
 "   and  has  plenty  of  options to customize dillo, so <STRONG>copy\n"
-"   it</STRONG>  to  your  <STRONG>~/.dillo/</STRONG>  directory, and\n"
+"   it</STRONG>  to  your  <STRONG>~/.etc/dillo/</STRONG>  directory, and\n"
 "   modify it to your taste.\n"
 " <li> Documentation for developers is in the <CODE>/doc</CODE>\n"
 "   dir  within  the  tarball;  you can find directions on everything\n"
